src/ConnectedMode.UnitTests/Binding/NonRoslynBindingConfigProviderTests.cs

```diff
@@ -37,26 +37,18 @@ public class NonRoslynBindingConfigProviderTests
     {
         private static readonly Language[] SupportedLanguages = { Language.C, Language.Cpp };
 
-        [TestMethod]
-        public void Ctor_NullSupportedLanguages_ArgumentNullException()
-        {
-            Action act = () => new NonRoslynBindingConfigProvider(null, Mock.Of<ISonarQubeService>(), Mock.Of<ILogger>());
-
-            act.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("supportedLanguages");
-        }
-
         [TestMethod]
         public void Ctor_NullService_ArgumentNullException()
         {
-            Action act = () => new NonRoslynBindingConfigProvider(SupportedLanguages, null, Mock.Of<ILogger>());
```

Comment:
Removed the `SupportedLanguages` argument since it is no longer required in the public constructor.